### DISCUSSION TITLE
How to fix 0.6.5 installation issue?

### DISCUSSION THREAD

**Discussion by ghost** [atoti/atoti#543] [Q&A] [answered]
Following the conda installation process on the atoti website, this issue happened

![image](https://user-images.githubusercontent.com/42935528/160787786-6156c813-71a0-466a-b549-d7c223278ca5.png)

I wiped my environment, followed the installation instructions and I now keep receiving this connection reset error (below). I’ve tried rebooting and deleting the package in question manually, but the error persists.

**Comment by GameTn** [comment]
Hi , have you tried this ?
pip install --upgrade atoti[jupyterlab] --user

**Comment by ghost** [comment]
Even if it's a conda install, you recommend to upgrade with pip ?

**Reply by GameTn** [reply]
Yes that's what i did in jupyterlab terminal and it works i did this first 
`pip install --upgrade atoti --user`
then
`pip install --upgrade atoti[jupyterlab] --user`

**Reply by tibdex** [reply]
Personally I don't recommend mixing `conda` with `pip`.

Also, a reliable way to install atoti is with Poetry, using the [Python package](https://docs.atoti.io/0.6.5/installation.html#python-package) instead of the Conda one.

**Comment by tibdex** [comment]
We just tried the following steps on a Windows machine with Conda 4.10.3 without issues:

- Create a new Conda environment
- Run `conda install atoti`

It correctly installed `jdk4py` and `atoti` 0.6.5.

The WARNING in your screenshot seems to indicate that the user doesn't not have full permissions on the `Anaconda3\pkgs` directory. Are they trying to install everything in their base Conda environment? Can they try with a new/empty Conda environment instead?

**Comment by ghost** [comment, marked as answer]
We can close this discussion as we think we have the explanation of the issue: seems related to a security feature making the package download time outing.
